src/very_good_dart_package/.github/workflows/main.yaml

```diff
@@ -15,3 +15,7 @@ jobs:
 
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
+{{#publishable}}
```

Comment:
Since this is not a compiled file, and this will have no effect on github because it is inside the `src` folder, I felt that it would be ok and easier to just add the template here.

Lmk what you think or if you prefer a different approach.

Reply:
Changed to the discussed approach!